### DISCUSSION TITLE
Text alignment.

### DISCUSSION THREAD

**Discussion by FatemaD1577** [jsvine/pdfplumber#1036] [Q&A]
Is there a way to find if a text line is center aligned?

**Comment by jsvine** [comment]
There is not a text property that explicitly indicates centering, but you could try `page.extract_text_lines(...)`, and then compare the `x0` and `x1` values to the page's bounding box (`page.bbox`). A center-aligned line would have its `x0`/`x1` edges equidistant from the edges of the page.

**Reply by FatemaD1577** [reply]
Thank you @jsvine. This seems to be helpful to some extent.
